p2p/libp2p/networksharding/noSharder.go

```diff
@@ -7,16 +7,16 @@ import (
 )
 
 // NoSharder default sharder, only uses Kademlia distance in sorting
-type noSharder struct {
+type NoSharder struct {
```

Comment:
Do you need this? Isn't this already implemented?

Reply:
We might need this in some tests where network sharding is not wanted